src/Notifier.js

```diff
@@ -192,12 +216,22 @@ var Notifier = {
         return enabled === 'true';
     },
 
-    setToolbarHidden: function(hidden) {
+     setToolbarHidden: function(hidden, persistent) {
         this.toolbarHidden = hidden;
         dis.dispatch({
             action: "notifier_enabled",
             value: this.isEnabled()
         });
+
+        if (persistent === true)
+            this.setToolbarPersistantHidden();
+    },
+
+    setToolbarPersistantHidden: function() {
```

Comment:
yes. it's only two lines, and only used in one place. Having a separate function doesn't do much for code clarity.